src/twisted/conch/test/test_agent.py

```diff
@@ -122,7 +126,7 @@ def test_agentc_REMOVE_ALL_RSA_IDENTITIES(self):
 
 if agent is not None:
 
-    class CorruptServer(agent.SSHAgentServer):
+    class CorruptServer(agent.SSHAgentServer):  # type: ignore[name-defined]
```

Comment:
Error here is:

```console
src/twisted/conch/test/test_agent.py:129:25: error: Name 'agent.SSHAgentServer' is not defined  [name-defined]
        class CorruptServer(agent.SSHAgentServer):
```

Reply:
I'm not sure why `mypy` finds this confusing, since we're in a block that excludes the case where `agent=None`.  Separating out the import of `keys` and `agent` didn't help.